lib/snippets.cson

```diff
@@ -28,10 +37,18 @@
     body: """
       '${1:.source.js}':
         '${2:Snippet Name}':
-          'prefix': '${3:hello}'
+          'prefix': '${3:Snippet Trigger}'
           'body': '${4:Hello World!}'$5
     """
 
+  'Atom Snippet Keys':
+    prefix: 'snipk'
+    body: """
+      '${1:Snippet Name}':
+        'prefix': '${2:Snippet Trigger}'
+        'body': '${3:Hello World!}'$5
```

Comment:
Should `'body': '${3:Hello World!}'$5` be `'body': '${3:Hello World!}'$4` ? (Notice the `$5` vs `$4`.)